ui/app/AppLayouts/Profile/stores/ContactsStore.qml

```diff
@@ -16,6 +18,15 @@ QtObject {
         Component.onCompleted: {
             mainModuleInst.resolvedENS.connect(root.resolvedENS)
         }
+
+        readonly property var conn: Connections {
+            target: d.contactsModuleInst
+            function onTrustStatusRemoved(pubKey: string) {
```

Comment:
QML JS supports argument typing?

Reply:
Well, yes and no :) In Qt 5, it's merely an annotation (still good as a hint); in Qt 6 it can be used for linting or even warnings/errors